exonum/src/runtime/types.rs

```diff
@@ -227,7 +227,18 @@ impl FromStr for ArtifactId {
 }
 
 /// Exhaustive artifact specification.
-#[derive(Debug, Clone, PartialEq, Eq, Hash, ProtobufConvert, BinaryValue, ObjectHash)]
+#[derive(
```

Comment:
Off-topic: Is it possible to do something to `rustfmt` for it to accept multi-line `derive`s like

```rust
#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
#[derive(ProtobufConvert, BinaryValue, ObjectHash)]
struct Foo { /* ... */ }
```

IMO, this would look significantly more readable.

Reply:
Unfortunately, no. We tried to split derives as you suggest, but `rustfmt` anyway prefer
this
very
beautiful
style.

Reply:
[This](https://rust-lang.github.io/rustfmt/#merge_derives) seems to be relevant.

Reply:
Strange. I remember I left comment here but it disappeared somehow...
Once more: I tried suggested approach, but it didn't help:
- `true` is default and it aligns averything horizontally but seems after some amount of elements it starts doing it vertically.
- If I set it to `false` and split big derives into two horizontal lines - they remain horizontally aligned but everything else becomes vertically aligned.

Reply:
I think we'll investigate it later, don't bother yourself.